fuel/transformers/__init__.py

```diff
@@ -65,21 +73,30 @@ def get_epoch_iterator(self, **kwargs):
         return super(Transformer, self).get_epoch_iterator(**kwargs)
 
     def get_data(self, request=None):
```

Comment:
(With the current logic, such an `AgnosticTransformer` would still require examples in, examples out or batches in, batches out, but I think that's fine.)